fuzzers/test_fuzzers.py

```diff
@@ -76,3 +79,14 @@ def test_fuzz_function_errors():
         # so fail if we see one. If that is not the case than this assert
         # should be removed.
         assert not isinstance(error.value, TypeError)
+
+
+def test_afl_get_stats(tmp_path):
```

Comment:
What do we think about putting the test for each fuzzer's get_stats here?
The more intuitive alternative is to create fuzzers/afl/test_fuzzer.py and put it there.
But I don't really love the idea of polluting fuzzer directories with this (and with test files as well).